README.md

```diff
@@ -95,9 +95,13 @@ repository. Reference the [issue](https://github.com/google/ground-android/issue
 
 ### Add Google Maps API Key(s)
 
-Create google_maps_api.xml files in gnd/src/debug/res/values and
-gnd/src/release/res/values, replacing API_KEY with debug and release Google Maps
-API keys:
+If you do not have them, generate *release* and *debug* Google Maps API keys by
+following the instructions at:
+
+  https://developers.google.com/maps/documentation/android-api/signup.
+
+In `gnd/src/debug/res/values` and `gnd/src/release/res/values` create
```

Comment:
Are these folders generated by gradle or android studio, or do devs have to create them? I ran through the initial API key setup flow in vim, so I just did a mkdir.

Reply:
I believe devs would need to create them.